tools/bwa.py

```diff
@@ -195,8 +215,9 @@ def mem(self, inReads, refDb, outAlign, options=None, min_score_to_filter=None,
         samtools = tools.samtools.SamtoolsTool()
 
         aln_sam = util.file.mkstempfname('.aligned.sam')
-        with samtools.bam2fq_tmp(inReads) as (fq1, fq2):
-            self.execute('mem', options + [refDb, fq1, fq2], stdout=aln_sam)
+        fastq_pipe = samtools.bam2fq_pipe(inReads)
```

Comment:
nice. Is this basically streaming to an interleaved fastq output (if paired)?

Reply:
Yup! Setting the`-p` flag for bwa mem allows for interleaved input.